pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -90,7 +90,6 @@ type ConverterContext struct {
 	HotplugVolumes        map[string]v1.VolumeStatus
 	PermanentVolumes      map[string]v1.VolumeStatus
 	DiskType              map[string]*containerdisk.DiskInfo
-	SRIOVDevices          map[string][]string
```

Comment:
I'm curious what's the motivation for removing this map from the context?

With this map, we had a separation where the convertor would simply use the information provided to it to construct a libvirt XML, while hardware-related operations, variables lookup, etc... would happen earlier in the manager part.

Reply:
The motivation is to keep everything close/localized when possible and not have things happening too far apart, making it harder to structure the code.

If we have a goal here not to cause the converter to depend on the env variables or the pod state, then it is not enforced already (in the converter we read now some fs paths and act on them).

What do you think about adding to the context the env variable raw information (or some basic rendering of it) and allowing the converter processing on it? I just want to avoid having the logic of iterating the vmi interfaces and based on that processing the env variables.
I prefer the information to be collected regardless the VMI spec and then in the converter to translate it to the domain correctly.
In later steps I hope to extract the network part out, and just use it from the converter. Then we can test it independently and re-struct it.

Does it make sense?